Dockerfile

```
@@ -112,22 +108,12 @@ RUN apt-get update && \
     && \
     apt-get clean
 
-RUN git clone --depth=1 --single-branch --branch=frankenphp-8.2 https://github.com/dunglas/php-src.git && \
-    cd php-src && \
-    #export CFLAGS="-DNO_SIGPROF" && \
-    # --enable-embed is only necessary to generate libphp.so, we don't use this SAPI directly
-    ./buildconf && \
-    ./configure \
-        --enable-embed=static \
-        --enable-zts \
-        --disable-zend-signals \
-        --with-config-file-path="$PHP_INI_DIR" \
-        --with-config-file-scan-dir="$PHP_INI_DIR/conf.d" && \
-    make -j$(nproc) && \
-    make install && \
-    rm -Rf php-src/ && \
-    ldconfig && \
-    php --version
+COPY --from=php-base /usr/local/include/php/ /usr/local/include/php
+COPY --from=php-base /usr/local/lib/libphp.* /usr/local/lib
```

```suggestion
COPY --from=php-base /usr/local/lib/libphp.a /usr/local/lib
```

(This will prevent loading a `.so` if any).